deployments/dev/environment.py

```diff
@@ -35,36 +35,38 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_DEPLOYMENT_STAGE': 'dev',
 
         'AZUL_CATALOGS': json.dumps({
-            name: dict(atlas=atlas,
-                       internal=internal,
-                       plugins=dict(metadata=dict(name='hca'),
-                                    repository=dict(name='tdr')))
-            for name, atlas, internal in [
-                ('dcp2', 'hca', False),
-                ('dcp2ebi', 'hca', False),
-                ('lungmap', 'lungmap', False),
-                ('it2', 'hca', True),
-                ('it2ebi', 'hca', True),
-                ('it3lungmap', 'lungmap', True)
-            ]
+            catalog_name: catalog_spec for name, atlas, sources, in (
+                (
+                    'dcp2',
+                    'hca',
+                    (
+                        'tdr:broad-jade-dev-data:snapshot/hca_dev_20201203___20210524_lattice:',
+                        'tdr:broad-jade-dev-data:snapshot/hca_dev_20210621_managedaccess_4298b4de92f34cbbbbfe5bc11b8c2422__20210622:'
+                    ),
+                ),
+                (
+                    'dcp2ebi',
+                    'hca',
+                    ('tdr:broad-jade-dev-data:snapshot/hca_dev_20201023_ebiv4___20210302:',)
+                ),
+                (
+                    'lungmap',
+                    'lungmap',
+                    ('tdr:broad-jade-dev-data:snapshot/lungmap_dev_20210412__20210414:',)
+                )
+            ) for catalog_name, catalog_spec in (
+                (name, dict(atlas=atlas,
```

Comment:
Just need to take this approach one step further:

```diff
Index: deployments/prod/environment.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/deployments/prod/environment.py b/deployments/prod/environment.py
--- a/deployments/prod/environment.py	(revision 0173461049c2f1949962d2225592f88c2293d7f6)
+++ b/deployments/prod/environment.py	(date 1631078543717)
@@ -40,39 +40,36 @@
         'AZUL_S3_BUCKET': 'edu-ucsc-gi-azul-dcp2-prod-storage-{AZUL_DEPLOYMENT_STAGE}',
 
         'AZUL_CATALOGS': json.dumps({
-            catalog_name: catalog_spec for name, atlas, sources, in (
+            f'{name}{suffix}': dict(atlas=atlas,
+                                    internal=False,
+                                    plugins=dict(metadata=dict(name='hca'),
+                                                 repository=dict(name='tdr')),
+                                    sources=sources)
+            for name, atlas, sources in [
                 (
                     'dcp8',
                     'hca',
-                    ('tdr:tdr-prd1-87360b54:snapshot/hca_prod_20201120_dcp2___20210812_dcp8:',)
+                    ['tdr:tdr-prd1-87360b54:snapshot/hca_prod_20201120_dcp2___20210812_dcp8:']
                 ),
                 (
                     'dcp9',
                     'hca',
-                    ('tdr:tdr-fp-99a334fd:snapshot/hca_prod_20201120_dcp2___20210827_dcp9',)
+                    ['tdr:tdr-fp-99a334fd:snapshot/hca_prod_20201120_dcp2___20210827_dcp9']
                 ),
                 (
                     'dcp1',
                     'hca',
-                    ('tdr:broad-datarepo-terra-prod-hca2:snapshot/hca_prod_20201118_dcp1___20201209:',)
+                    ['tdr:broad-datarepo-terra-prod-hca2:snapshot/hca_prod_20201118_dcp1___20201209:']
                 ),
                 (
                     'lungmap',
                     'lungmap',
-                    ('tdr:broad-datarepo-terra-prod-hca2:snapshot/hca_prod_20210616_lungmap___20210616_lm1:',)
+                    ['tdr:broad-datarepo-terra-prod-hca2:snapshot/hca_prod_20210616_lungmap___20210616_lm1:']
                 )
-            ) for catalog_name, catalog_spec in (
-                (name, dict(atlas=atlas,
-                            internal=False,
-                            plugins=dict(metadata=dict(name='hca'),
-                                         repository=dict(name='tdr')),
-                            sources=sources)),
-                (f'{name}-it', dict(atlas=atlas,
-                                    internal=True,
-                                    plugins=dict(metadata=dict(name='hca'),
-                                                 repository=dict(name='tdr')),
-                                    sources=sources))
-            )
+            ] for suffix, internal in [
+                ('', False),
+                ('-it', True)
+            ]
         }),
         'AZUL_TDR_SERVICE_URL': 'https://jade-terra.datarepo-prod.broadinstitute.org',
         'AZUL_SAM_SERVICE_URL': 'https://sam.dsde-prod.broadinstitute.org',
```

Also helps to mix in a list here and there to lighten up the nesting.